examples/code_with_line_highlights.html

```diff
@@ -0,0 +1,7 @@
+<pre data-line="3, 5-6"><code class="language-python">
```

Comment:
We're not adding in anything for the `line_start` here - either delete it from the markdown, or use the `data-line-offset` thing we were looking at.

Reply:
line_start is only used in conjunction with the `line_numbers` setting instead. I should be testing it with that, so I'll add it in to the relevant test.